test/petab/test_petabSimulator.py fails

Possibly related to the recent basico release (https://pypi.org/project/copasi-basico/#history):


```
______________________ test_petab_case[0016-sbml-v1.0.0] _______________________

case = '0016', model_type = 'sbml', version = 'v1.0.0'

    @pytest.mark.parametrize(
        "case, model_type, version",
        [
            (case, "sbml", "v1.0.0")
            for case in petabtests.get_cases(format_="sbml", version="v1.0.0")
        ],
    )
    def test_petab_case(case, model_type, version):
        """Wrapper for _execute_case for handling test outcomes"""
        try:
            _execute_case(case, model_type, version)
        except Exception as e:
            if isinstance(
                e, NotImplementedError
            ) or "Timepoint-specific parameter overrides" in str(e):
                logger.info(
                    f"Case {case} expectedly failed. Required functionality is "
                    f"not implemented: {e}"
                )
                pytest.skip(str(e))
            else:
>               raise e

test/petab/test_petabSimulator.py:37: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
test/petab/test_petabSimulator.py:26: in test_petab_case
    _execute_case(case, model_type, version)
test/petab/test_petabSimulator.py:64: in _execute_case
    simulator = basico.petab.PetabSimulator(petab_problem)
.tox/petab/lib/python3.10/site-packages/basico/petab/core.py:282: in __init__
    self.model = self.load_model(**kwargs)
.tox/petab/lib/python3.10/site-packages/basico/petab/core.py:302: in load_model
    self.cps_file = load_petab(files['problem'], self.working_dir)
.tox/petab/lib/python3.10/site-packages/basico/petab/core.py:235: in load_petab
    converter.convert()
.tox/petab/lib/python3.10/site-packages/copasi_petab_importer/convert_petab.py:863: in convert
    self.generate_copasi_file(self.petab, self.out_dir, self.out_name)
.tox/petab/lib/python3.10/site-packages/copasi_petab_importer/convert_petab.py:816: in generate_copasi_file
    if not dm.importSBMLFromString(petab.transformed_sbml):
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <CDataModel "Root">
sbmlDocumentText = '<?xml version="1.0" encoding="UTF-8"?>\n<sbml xmlns="http://www.sbml.org/sbml/level2/version4" level="2" version="4">...  </apply>\n          </math>\n        </kineticLaw>\n      </reaction>\n    </listOfReactions>\n  </model>\n</sbml>\n'
pProcessReport = None, deleteOldData = 1, importMiriam = True
importInitialValues = False

    def importSBMLFromString(self, sbmlDocumentText, pProcessReport=None, deleteOldData=1, importMiriam=True, importInitialValues=False):
        r"""importSBMLFromString(CDataModel self, std::string const & sbmlDocumentText, CProcessReport pProcessReport=None, bool const & deleteOldData=1, bool importMiriam=True, bool importInitialValues=False) -> bool"""
>       return _COPASI.CDataModel_importSBMLFromString(self, sbmlDocumentText, pProcessReport, deleteOldData, importMiriam, importInitialValues)
E       TypeError: Wrong number or type of arguments for overloaded function 'CDataModel_importSBMLFromString'.
E         Possible C/C++ prototypes are:
E           CDataModel::importSBMLFromString(std::string const &,CProcessReport *,bool const &,bool,bool)
E           CDataModel::importSBMLFromString(std::string const &,CProcessReport *,bool const &,bool)
E           CDataModel::importSBMLFromString(std::string const &,CProcessReport *,bool const &)
E           CDataModel::importSBMLFromString(std::string const &,CProcessReport *)
E           CDataModel::importSBMLFromString(std::string const &)

.tox/petab/lib/python3.10/site-packages/COPASI/__init__.py:18721: TypeError
```

See https://github.com/ICB-DCM/pyPESTO/actions/runs/14634447500/job/41062584642

--- a/tox.ini
+++ b/tox.ini
@@ -87,9 +87,6 @@ commands =
     python3 -m pip install git+https://github.com/PEtab-dev/petab_test_suite@main
     python3 -m pip install git+https://github.com/pysb/pysb@master
     python3 -m pip install -U copasi-basico[petab]
-    # https://github.com/ICB-DCM/pyPESTO/issues/1569
-    # https://github.com/copasi/basico/issues/68
-    python3 -m pip install -U python-copasi==4.45.296
     # upgrade after installing pysb which requires an older sympy
     python3 -m pip install -U sympy
     pytest --cov=pypesto --cov-report=xml --cov-append \

